app/Controllers/Http/AuthController.ts

```diff
@@ -0,0 +1,42 @@
+import UserValidator from "App/Validators/UserValidator";
+import User from "App/Models/User";
+
+class AuthController {
+  public async register({ request, response }) {
+    try {
+      const { email, password } = await request.validate(UserValidator);
+
+      const user = await User.create({
+        email,
+        password,
+      });
+
+      return response.status(201).json(user);
```

Comment:
are we sending password in response, if yes then do not send password in response.
or we can just send `{status: true, message: user registered successfully}`